terraform/workspace_variables/production.tfvars.json

```
@@ -11,8 +11,8 @@
   "forms_container_delete_retention_days": 30,
   "statuscake_alerts": {
     "tra-apply-for-qts-prod": {
-      "website_name": "apply-for-qts-in-england-production",
-      "website_url": "https://apply-for-qts-in-england-production.london.cloudapps.digital/healthcheck",
+      "website_name": "apply-for-qts-in-england.education.gov.uk",
+      "website_url": "https://apply-for-qts-in-england.education.gov.uk/eligibility/start/healthcheck",
```

```suggestion
      "website_url": "https://apply-for-qts-in-england.education.gov.uk/healthcheck",
```

This should be the URL for the health check.

---

fixed